ampersand-form-view.js

```diff
@@ -31,6 +31,12 @@ module.exports = View.extend({
         }
     },
 
+    template: [
+        '<form>',
+            '<fieldset data-hook="field-container"></fieldset>',
```

Comment:
i'm not opposed to a fieldset.  i would consider this a breaking change however, as any DOM node type or hierarchy manipulations will impact folks' CSS rules.  it may also be a nice addition for some people to have more immediate structure.  the counter to this is that it's not strictly necessary/bare-minimum, and is easily overridden with a template